index.js

```diff
@@ -48,7 +50,7 @@ function add(id, label = id) {
   container.add(id, {
     level: LEVEL,
     format: defaultFormat(label),
-    transports: [new ConsoleTransport({ silent: TEST })],
+    transports: [consoleTransport],
```

Comment:
so it's ok to have the same transport for each container?

Reply:
yeah, according to the issues i dug up /shrug hoping v3 eventually adds a better api here.